src/actions/websocket.js

```diff
@@ -212,9 +222,10 @@ function handleEvent(msg) {
     case WebsocketEvents.UPDATE_TEAM:
         doDispatch(handleUpdateTeamEvent(msg));
         break;
-    case WebsocketEvents.PATCH_TEAM:
-        doDispatch(handlePatchTeamEvent(msg));
-        break;
+
+    // case WebsocketEvents.PATCH_TEAM:
+    //     doDispatch(handlePatchTeamEvent(msg));
+    //     break;
```

Comment:
There is no `WebsocketEvents.PATH_TEAM` constant present. I commented out the case. But it is rather needs to be removed. Comments welcome

Reply:
Please remove it.

Reply:
The code is removed. Now resolving the conversation.